src/gen-def.py

```diff
@@ -7,7 +7,7 @@
 headers_content = []
 for h in os.environ["headers"].split (' '):
 	if h.endswith (".h"):
-		with io.open(h, encoding='utf8') as f: headers_content.append (f.read ())
+		with io.open (h, encoding='ISO-8859-1') as f: headers_content.append (f.read ())
```

Comment:
Can you explain why this is needed? Why not ASCII? Our headers should be UTF-8. latin1 definitely doesn't sound like has any place here.

Reply:
I know that this doesn't make sense but a msys2 bot didn't like that.

Reply:
try to print the name of the header that was failing with UnicodeDecodeError when being read as UTF-8, and make sure it is indeed encoded as UTF-8.

Reply:
The issue is `glib-mkenums` generated `hb-gobject-enums.h` which is in ASCII on msys2 apparently but I am not sure about the fix